scripts/default_mpnn_020.sh

```
@@ -0,0 +1,20 @@
+#SBATCH --partition=volta-gpu
+#SBATCH --cpus-per-task=12
+#SBATCH --mem=128G
+#SBATCH --gres=gpu:1
+#SBATCH --output=R-%x.%j.out
+#SBATCH --error=R-%x.%j.err
+#SBATCH --qos=gpu_access
+
+module load gcc/11.2.0
+module load cuda/11.8
+
+cd /proj/kuhl_lab/users/dieckhau/ThermoMPNN/proteinmpnn/ 
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)